_posts/code/Canny/canny-run.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -o xtrace
+
+wget --no-clobber https://johnysswlab.com/downloads/221575.tar.gz
```

Comment:
This is not needed since you uploaded the sources already, right?

Reply:
It is needed. The file is huge (5 MB). Normally I don't keep large binary files in the repository, but if you want, I can upload it here too

Reply:
Oh, it's an input file. Then OK.